### PR TITLE
Skeleton components for each pages

### PR DESCRIPTION
**Pull request by nkrulikovsky**
- Skeleton of pages
- Dark and light theme update for storybook component
- Update page folder structure.

Screen Shot 2022-10-17 at 7 45 34 PM
Screen Shot 2022-10-17 at 7 45 43 PM
image

Screen Shot 2022-10-17 at 7 46 35 PM

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ module.exports = {
   addons: [
     "@storybook/addon-links",
     "@storybook/addon-essentials",
-    "@storybook/preset-create-react-app",
+    // "@storybook/preset-create-react-app",
```
Comment (hammeiam) [inline .storybook/main.js:6]: Should this be deleted or kept?

**Review by hammeiam — APPROVED** [review]

File: .storybook/main.js (hunk 1 above)
Comment (nkrulikovsky) [reply .storybook/main.js:6]: We can delete.

File: .storybook/story-layout.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  const muiDarkTheme = darkTheme
+  muiDarkTheme.components = componentsOverrides(muiDarkTheme)
+  const [isLightTheme, setIsLightTheme] = useState(false)
+  const currentTheme = isLightTheme ? muiLightTheme : muiDarkTheme
+
+  const handleChange = (event) => {
+    setIsLightTheme(event.target.checked)
+  }
+
+  console.log("first", currentTheme.palette.background.paper)
```
[the remark this answers is not in the document]
Comment (nkrulikovsky) [reply .storybook/story-layout.js]: removed

File: .storybook/main.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (nkrulikovsky) [reply .storybook/main.js]: Deleted!

**Pull request merged**